README.md

```diff
@@ -94,3 +94,14 @@ Write SAM files from `Sam` objects
 >>> out_sam.write(x)
 >>> out_sam.close()
 ```
+
+Write SAM files from `Sam` objects to stdout (allows `samtools view` compression)
```

Comment:
Great idea to add an example to the documentation. I like this.

Reply:
Glad to hear that